website/content/v1.10/kubernetes-guides/configuration/inlinemanifests.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "inlineManifests and extraManifests"
+description: "Learn what inlineManifests and extraManifests are, how they differ, and why they matter."
+---
+
+`inlineManifests` and `extraManifests` allow you to automatically apply Kubernetes resources to your cluster during installation and upgrades.
+
+Both are designed to automate the provisioning of components like CNIs and other static infrastructure, but they differ in how the manifest content is sourced and applied.
+They are not meant for deploying applications or frequently changing services.
+For those, it's better to use a GitOps or CI/CD tool.
+
+## inlineManifests
+
+`inlineManifests` are defined directly within the machine configuration file.
+The YAML content is embedded inside the `inlineManifests` section, making it ideal for tightly coupled resources that need to be provisioned as soon as the node boots up.
+
+Here’s an example of how to configure a cluster using an `inlineManifest`:
+
+```yaml
+cluster:
+  inlineManifests:
+    - name: my-app
+      contents: |
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: my-application
+        spec:
+          # ... deployment specification
```

Comment:
Would it be useful to explain that despite being an array, you cannot rely on resources being applied in order. However you can use multi-doc syntax `---` if necessary

Reply:
Yes, like https://www.talos.dev/v1.10/kubernetes-guides/network/deploying-cilium/#method-5-using-a-job